fpga_interchange/rapidyaml_support.py

```diff
@@ -91,6 +91,8 @@ def make_reference(self, ref_id):
             return YamlReference(ref)
 
     def get_list_id(self, field, index):
+        if field not in self.field_ids.keys():
+            return 0
```

Comment:
Can you elaborate on why this change is required ?

Reply:
This change fixes invalid key if capnp file didn't contain some fields. I think that this broke for device schema.